src/babel.dev.js

```diff
@@ -27,13 +30,15 @@ module.exports = function plugin(args) {
   }
   const { types: t, template } = args
 
+  const packageAbsolutePath = module.exports.getPackageAbsolutePath()
```

Comment:
Yes, that's what I did at the very first. Then I found the snapshot test failed because `__dirname` varies in different environment.

![](https://img11.360buyimg.com/uba/jfs/t26707/93/2300069595/62262/f72587af/5bfe079aNaff04821.png)

So I make it and exports that I can mock it in the tests.

Any suggestion on how to do it in a better way? :)